backend/src/routes/api/newsletter/index.ts

```diff
@@ -23,28 +23,33 @@ router.get("/get_newsletters", VerifyToken, async(req, res)=>{
     }
 });
 
-
-router.post("/create_newsletter", async(req, res)=>{
-    // Parse through the text in here
-    var titleText = req.body.title;
-    var bodyText = req.body.body;
-    // var author = req.body.author;
-    if(!titleText || !bodyText){
-        res.status(400).send(JSON.stringify("Bad user input. Inputs required for all fields."));
-        return;
-    }
-    console.log('Got a POST request for the homepage for this: ');
-    // Create a newsletter item in the database here
-    try {
-        const newsletterItem = await Newsletter.create({
-            title: titleText, // change this based on what the req looks like
-            body: bodyText,
-            // author: author
-        })
-        console.log("Successfully added to the database: " + newsletterItem)
-    } catch(e) {
-        console.log(e.message)
-    }
+router.post("/create_newsletter", async (req, res) => {
+  // Parse through the text in here
+  var titleText = req.body.title;
+  var bodyText = req.body.body;
+  // var author = req.body.author;
+  if (!titleText || !bodyText) {
+    res
+      .status(400)
+      .send(JSON.stringify("Bad user input. Inputs required for all fields."));
+    return;
+  }
+  console.log("Got a POST request for the homepage for this: ");
+  // Create a newsletter item in the database here
+  try {
+    const newsletterItem = await Newsletter.create({
+      title: titleText, // change this based on what the req looks like
+      body: bodyText,
+      // author: author
+    });
+    res.status(200).send({
+      message: `Successfully added to the database: ${newsletterItem}`
+    });
+  } catch (e) {
+    res.status(400).send({
+      message: `Failed to add this post to the database. ${e}`
+    });
+  }
 });
 
 router.delete('/delete_newsletter', async(req, res)=>{
```

Comment:
could you add a comment or something here as to check if the user owns the newsletter they are about to delete. We dont want user's to be able to delete any arbitrary newsletter...